docs/getting-started/github-apps/github-apps-landing-page.rst

```diff
@@ -8,22 +8,20 @@ Users can grant a GitHub app specific permissions on the repos and/or organizati
 The Dockstore GitHub App adds support for registering and automatically syncing workflows and services.
```

Comment:
I would flip the content at the beginning of this page and rephrase a bit:

The Dockstore GitHub App adds support for registering and automatically syncing tools, workflows, and services hosted on GitHub. 

GitHub apps are a GitHub feature used to improve the interaction between external applications and GitHub. Users can grant a GitHub app specific permissions on the GitHub repos and/or organizations of their choosing.

Check out the following guides to learn more: